projects/plugins/jetpack/extensions/shared/memberships/subscribers-affirmation.js

```diff
@@ -141,8 +142,8 @@ export const getCopyForSubscribers = ( {
 		return sprintf(
```

Comment:
I don't think this is from your changes, but this block and the one below have the same condition but different comments: `if ( isPaidPost && ! postHasPaywallBlock ) {`

Reply:
Nice catch. Will remove the 2nd if block.

Reply:
Addressed in https://github.com/Automattic/jetpack/pull/43193/commits/d531867b33817e7bafa9cb9aca5712e408a0de2f.